cmd/run_scenario.go

```diff
@@ -47,6 +49,7 @@ func (r *scenarioRunner) addCLIFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&r.runID, "run-id", "", "Run ID for this run")
 	fs.IntVar(&r.iterations, "iterations", 0, "Iterations for the scenario (cannot be provided with duration)")
 	fs.DurationVar(&r.duration, "duration", 0, "Duration for the scenario (cannot be provided with iteration)")
+	fs.DurationVar(&r.waitForServer, "wait-for-server", 0, "Duration to try to connect to server before failing")
```

Comment:
`connect-timeout` maybe? `wait-for-server` sounds to me like a condition (boolean) not a duration.
I'd accept another name where it's clear that this is a duration.

Reply:
:+1: Will fix in next PR coming shortly after this one.